github/actions_workflow_jobs.go

```diff
@@ -38,6 +38,8 @@ type WorkflowJob struct {
 	Name        *string     `json:"name,omitempty"`
 	Steps       []*TaskStep `json:"steps,omitempty"`
 	CheckRunURL *string     `json:"check_run_url,omitempty"`
+	// Labels is runner labels from the `runs-on:` key from a GitHub Actions workflow
```

Comment:
```suggestion
	// Labels represents runner labels from the `runs-on:` key from a GitHub Actions workflow.
```

(This is a minor nit tweak, so I'll go ahead and commit it... nothing necessary to do on your end.)